<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidgetUI</class>
 <widget class="QWidget" name="MainWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <widget class="QLabel" name="IconLabelShow">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>10</y>
     <width>180</width>
     <height>180</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CodeLabel">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>270</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>CODE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>445</x>
     <y>370</y>
     <width>35</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CodeLineEditShow">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>428</width>
     <height>530</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="ContentWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>426</width>
      <height>528</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="NameLineEditShow">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
